LibGUI/GAbstractButton.h

```diff
@@ -49,11 +49,11 @@ class GAbstractButton : public GWidget {
     virtual bool is_abstract_button() const final { return true; }
 
     String m_text;
-    bool m_checked { false };
-    bool m_checkable { false };
-    bool m_hovered { false };
-    bool m_being_pressed { false };
-    bool m_exclusive { false };
+    bool m_checked{ false };
```

Comment:
Hmm.. What version of `clang-format` are you using? It's not supposed to be doing this.
I've tested with 7.0.0 and 8.0.1 here, and both keep the old style.

Reply:
I'm on 6, which is the default for Ubuntu.

Reply:
Okay, please use a newer version that understands our style :)